dev/bump_version.json

```
@@ -0,0 +1,11 @@
+{
+    "files": {
+        "setup.py": "version=\"$VERSION\"",
+        "mwdb/version.py": "app_version = \"$VERSION\"",
+        "mwdb/web/package.json": "\"version\": \"$VERSION\"",
+        "mwdb/web/package-lock.json": "\"version\": \"$VERSION\"",
+        "mwdb/web/src/commons/package.json": "\"version\": \"$VERSION\"",
+        "docs/conf.py": "release = '$VERSION'"
+    },
+    "regex": "(\\d.\\d.\\d(?:-(post|dev|alpha|beta|rc)[0-9])?)"
```

Allow multidigit versions and escape dot
```suggestion
    "regex": "(\\d+\\.\\d+\\.\\d+(?:-(post|dev|alpha|beta|rc)[0-9])?)"
```

---

Fixed `.` => `[.]`